.github/workflows/main.yml

```diff
@@ -61,7 +62,13 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: helm-artifacts
-          path: ${{ github.workspace }}/helm-charts
+          path: ./helm-charts
```

Comment:
Ensure that the path `./helm-charts` is correct and that the Helm charts are available at this location during the workflow execution. This is crucial for uploading the Helm artifacts.